scripts/ci/upload-autodoc.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2019 Tocqueville Group
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+set -e -o pipefail
+
+git config --global user.email "hi@serokell.io"
+git config --global user.name "CI autodoc generator"
+git remote remove auth-origin 2> /dev/null || :
+git remote add auth-origin https://serokell:$GITHUB_TOKEN@github.com/tezos-commons/homebase-lite.git
```

Comment:
> we need to get the password into the buildkite's pipeline somehow

You are right, I thought it was there already, but it wasn't. It is now but [there seems to be another problem](https://buildkite.com/serokell/homebase-lite/builds/96#e1d4915a-31e4-4fe1-b92b-226c0e7fc4b8/78-86).

As you can see from the logs `Found environment at serokell/homebase-lite/environment (63 bytes)` the token is now found and the error message is different.

I suspect (simply because it was the case elsewhere) that quoting the variable may work:
```suggestion
git remote add auth-origin https://serokell:"$GITHUB_TOKEN"@github.com/tezos-commons/homebase-lite.git
```
but I may be wrong.

Reply:
[Uh...](https://buildkite.com/serokell/homebase-lite/builds/106#118d9073-1b4d-418e-b7de-c1a22ddba3af/83)

```
  | remote: Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.
  | remote: Please see https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information.
  | fatal: Authentication failed for 'https://github.com/tezos-commons/homebase-lite.git/'
```

Reply:
Sorry, I put the wrong token in the vault. It works now :see_no_evil:

Reply:
Cool.